src/stores/prompts.js

```diff
@@ -247,35 +282,80 @@ export const usePromptStore = defineStore('prompts', {
       return getDownloadURL(ref(storage, storageRef))
     },
 
-    async addLike(id) {
-      this._prompts = []
-      this.$patch({ _prompts: this.getPrompts })
-
+    async addLike(promptId) {
       this._isLoading = true
-      await updateDoc(doc(db, 'prompts', id), {
-        'info.likes': arrayUnion(useUserStore().getUserRef),
-        'info.dislikes': arrayRemove(useUserStore().getUserRef)
-      })
-        .then(() => this.fetchPromptById(id))
-        .catch((error) => {
-          console.error(error)
-          throw new Error(error)
+      let browserId
+      await useUserStore()
+        .loadBrowserId()
+        .then(() => {
+          browserId = useUserStore().getBrowserId
         })
```

Comment:
Is the regular way of using state management not working here?
```suggestion
      await useUserStore().loadBrowserId()
      const browserId = useUserStore().getBrowserId
```

Reply:
Yes i agree. 
I was try somes stuff, so i forget to improve the code base.